src/script/components/MessagesList/Message/ContentMessage/asset/TextMessageRenderer.tsx

```diff
@@ -67,97 +67,48 @@ export const TextMessageRenderer: FC<TextMessageRendererProps> = ({
       return undefined;
     }
     const emailLinks = containerRef.current && [...containerRef.current.querySelectorAll('[data-email-link]')];
-    const linkTargets = containerRef.current && [...containerRef.current.querySelectorAll('a[data-md-link]')];
-    const msgLinkTargets = containerRef.current && [...containerRef.current.querySelectorAll('[data-uie-name]')];
+    const markdownLinkTargets = containerRef.current && [...containerRef.current.querySelectorAll('[data-md-link]')];
     const hasMentions = asset && asset.mentions().length;
     const msgMention = hasMentions
       ? containerRef.current && [...containerRef.current.querySelectorAll('.message-mention')]
```

Comment:
```suggestion
      ? [...containerRef.current.querySelectorAll('.message-mention')]
```

You don't need to check for containerRef here, you are already doing it earlier in the code :)